app/views/content/terms_candidate.md

```diff
@@ -1,40 +1,53 @@
+If you apply for a course or accept an offer on both, we’ll get in touch with you to cancel one of your applications.
 
-[Get Into Teaching](https://getintoteaching.education.gov.uk/get-help-and-support) can guide you through the process of applying for more courses. Contact them for free on 0800 389 2500, or [chat to an adviser using the online chat service](https://getintoteaching.education.gov.uk/lp/live-chat).
+### Stage 2: Apply again if your application was not successful
+
+You can apply for another course at this stage if you do not get an offer or if you choose not to accept your offers.
 
+[Get Into Teaching](https://getintoteaching.education.gov.uk/get-help-and-support) can guide you through the process of applying for more courses. Contact them for free on 0800 389 2500, or [chat to an adviser using the online chat service](https://getintoteaching.education.gov.uk/lp/live-chat).
+ 
 ### Making changes to your application
 
-Get in touch with us as soon as possible after submitting your application if you need to make any amends, as we only have 5 working days to make changes for you.
+Contact us at <becomingateacher@digital.education.gov.uk> if you need to make changes. We only have 5 working days to make changes for you. 
 
-For example, you can add more courses (if you have not reached your limit of 3) or change your course choices.
+Once we’ve processed your request, we will not be able to make any more changes.
 
-Once we’ve processed your request, we will not be able to make any more changes for you.
+You can add more courses (if you have not reached your limit of 3) or change the courses you have chosen.
 
-However, you can ask us to amend your name or contact details at any point up until enrolment.
+You can ask us to amend your name or contact details at any point until you are enrolled on a course.
 
-Contact us at <becomingateacher@digital.education.gov.uk> if you need to make changes.
+If you have heard back from your training provider, for example if they have turned down your application, we will not be able to change your choice of course.
 
 ### Withdrawing your application
 
-You can withdraw your application to your course(s) at any point, even after you’ve accepted an offer.
+You can withdraw your application at any point, even after you have accepted an offer.
+
+[Sign in to your account](https://qa.apply-for-teacher-training.service.gov.uk/candidate/sign-in) and click ‘withdraw’ next to the course you want to withdraw.
 
-[Sign in to your account](/candidate/sign-in) and click ‘withdraw’ next to the course(s) you want to withdraw.
+### Outcome of your application
+
+The Department of Education does not decide on your application. The course provider does.
+
+There are deadlines that you and the course provider will need to meet. We’ll tell you what they are when you apply. If you, or the course provider, don’t respond in time your application might be deleted.
```

Comment:
We should avoid negative contractions for consistency.

Change 'don't' to 'do not'